notion/renderer.py

```diff
@@ -0,0 +1,241 @@
+			id="chk_" + block.id,
+			checked=" checked" if block.checked else "",
+			title=block.title,
+		)
+
+	def handle_code(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "code", attributes=self.left_margin_for_level(level))
+
+	def handle_factory(self, block, level=0, preblock=None, postblock=None):
+		return ""
+
+	def handle_header(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "h2", attributes=self.left_margin_for_level(level))
+
+	def handle_sub_header(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "h3", attributes=self.left_margin_for_level(level))
+
+	def handle_sub_sub_header(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "h4", attributes=self.left_margin_for_level(level))
+
+	def handle_page(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "h1", attributes=self.left_margin_for_level(level))
+
+	def handle_bulleted_list(self, block, level=0, preblock=None, postblock=None):
+		text = ""
+		if preblock is None or preblock.type != "bulleted_list":
+			text = self.create_opening_tag("ul", attributes=self.left_margin_for_level(level))
+		text += self.wrap_in_tag(block, "li")
+		if postblock is None or postblock.type != "bulleted_list":
+			text += "</ul>"
+		return text
+
+	def handle_numbered_list(self, block, level=0, preblock=None, postblock=None):
+		text = ""
+		if preblock is None or preblock.type != "numbered_list":
+			text = self.create_opening_tag("ol", attributes=self.left_margin_for_level(level))
+		text += self.wrap_in_tag(block, "li")
+		if postblock is None or postblock.type != "numbered_list":
+			text += "</ol>"
+		return text
+
+	def handle_toggle(self, block, level=0, preblock=None, postblock=None):
+		innerhtml = markdown2.markdown(block.title)
+		opentag = self.create_opening_tag("details", attributes=self.left_margin_for_level(level))
+		return '{opentag}<summary>{innerhtml}</summary>'.format(opentag=opentag, innerhtml=innerhtml), '</details>'
+
+	def handle_quote(self, block, level=0, preblock=None, postblock=None):
+		return self.wrap_in_tag(block, "blockquote", attributes=self.left_margin_for_level(level))
+
+	def handle_text(self, block, level=0, preblock=None, postblock=None):
+		return self.handle_default(block=block, level=level, preblock=preblock, postblock=postblock)
+
+	def handle_equation(self, block, level=0, preblock=None, postblock=None):
+		text = self.create_opening_tag("p", attributes=self.left_margin_for_level(level))
+		return text + '<img src="https://chart.googleapis.com/chart?cht=tx&chl={}"/></p>'.format(block.latex)
+
+	def handle_embed(self, block, level=0, preblock=None, postblock=None):
+		iframetag = self.create_opening_tag("iframe", attributes={
+			"src": block.display_source or block.source,
+			"frameborder": 0,
+			"sandbox": "allow-scripts allow-popups allow-forms allow-same-origin",
```

Comment:
Are these defaults for embed `sandbox` sourced from somewhere/based on a best practice/needed for some of the default embeds that notion supports?